tests/unity/unity_internals.h

```diff
@@ -684,15 +684,15 @@ UNITY_INTERNAL_PTR UnityDoubleToPtr(const double num);
  * Error Strings We Might Need
  *-------------------------------------------------------*/
 
-extern const char UnityStrOk[];
-extern const char UnityStrPass[];
-extern const char UnityStrFail[];
-extern const char UnityStrIgnore[];
-
-extern const char UnityStrErrFloat[];
-extern const char UnityStrErrDouble[];
-extern const char UnityStrErr64[];
-extern const char UnityStrErrShorthand[];
+LV_ATTRIBUTE_EXTERN_DATA extern const char UnityStrOk[];
```

Comment:
Tests are not compiled to a shared library so no need to add `LV_ATTRIBUTE_EXTERN_DATA` here.